index.php

```diff
@@ -20,6 +20,7 @@
     ENDPOINT => function() {
 
         header('Access-Control-Allow-Origin: *');
+        header('Access-Control-Allow-Headers: content-type');
```

Comment:
Thank you for merging. Quite good answer is available [here].(http://stackoverflow.com/questions/10636611/how-does-access-control-allow-origin-header-work).
The framework that I use just send non-simple request and I need this flag.